packages/amplify-cli/package.json

```diff
@@ -99,5 +99,94 @@
       "node"
     ],
     "collectCoverage": true
+  },
+  "amplify": {
```

Comment:
Why do we need to add this to our main package.json? Each time we add a plugin, we need to add a blog here?

Reply:
It was originally put inside the health check ts file, but Attila and Yathi suggested to put it here, and I do agree with them because it is easier to check for consistency. If we add an official plugin, the CLI core will reference it as a dependency, and also put its information inside the amplify field in the package.json.

Reply:
We need to document this behavior in some docs otherwise it's super hard to remember the existence of new blocks and requirements like these.

Reply:
Please add a section to the plugins doc explaining this section. Thanks!